jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/kafka/OdeKafkaProperties.java

```diff
@@ -3,14 +3,31 @@
 import lombok.Data;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.validation.annotation.Validated;
+import us.dot.its.jpo.ode.wrapper.MessageProducer;
 
 import java.util.Set;
 
 @Configuration
 @ConfigurationProperties(prefix = "ode.kafka")
 @Data
+@Validated(value = OdeKafkaPropertiesValidator.class)
 public class OdeKafkaProperties {
     private String brokers;
-    private String producerType;
     private Set<String> disabledTopics;
+    private Producer producer;
+
+    @Data
+    public static class Producer {
+        private Integer batchSize = MessageProducer.DEFAULT_PRODUCER_BATCH_SIZE_BYTES;
+        private Integer bufferMemory = MessageProducer.DEFAULT_PRODUCER_BUFFER_MEMORY_BYTES;
+        private Integer lingerMs = MessageProducer.DEFAULT_PRODUCER_LINGER_MS;
+        private Integer retries = MessageProducer.DEFAULT_PRODUCER_RETRIES;
+        private String acks = MessageProducer.DEFAULT_PRODUCER_ACKS;
+        private String keySerializer = MessageProducer.SERIALIZATION_STRING_SERIALIZER;
+        private String valueSerializer = MessageProducer.SERIALIZATION_BYTE_ARRAY_SERIALIZER;
+        private String compressionType = MessageProducer.COMPRESSION_TYPE;
```

Comment:
note: I considered moving these defaults from [MessageProducer](https://github.com/CDOT-CV/jpo-ode/blob/3bd0f7717575fc04508f6fe3c85ee632878ad8d6/jpo-ode-core/src/main/java/us/dot/its/jpo/ode/wrapper/MessageProducer.java#L38) into this class, but that would cause a dependency cycle between` jpo-ode-svcs` (this module) and `jpo-ode-core` (MessageProducers' module). Once we migrate to Spring Kafka, we won't need `OdeKafkaProperties` nor `MessageProducer` anymore, so once I saw the cycle, I decided to leave the code as it is because it doesn't have much longer to live anyway.